app/org/views.py

```diff
@@ -0,0 +1,31 @@
+from flask import render_template
+from ..models import EditableHTML
+from flask.ext.login import (
+    login_required,
+    login_user,
+    logout_user,
+    current_user
+)
+from . import main
+
+
+
+@org.route('/')
```

Comment:
Oh nvm my previous comment, but you need to create a `__init__.py` file creating the `org` blueprint. Additionally, you'll need to edit `__init__.py` under the `app` folder (i think thats the filename), to mount the blueprint.

Reply:
done!